pom.xml

```diff
@@ -2064,6 +2065,19 @@
                         </dependency>
                     </dependencies>
                 </plugin>
+                <!-- we need a separate enforcer configuration to append new rules rather then override existing ones -->
```

Comment:
This will cause warnings when Maven is run. I think we could solve this by using the [rule groups](https://github.com/skuzzle/restrict-imports-enforcer-rule#rule-groups) feature instead of adding top level rules.

However, I'm not sure that this import restriction is necessary, as we already ban the dependency. Using it as an import will result in a dependency analyze failure.

Reply:
I rebased locally and see this error for an import that was added later:
```
INFO] --- dependency:3.5.0:analyze-only (default) @ trino-iceberg ---
[ERROR] Used undeclared dependencies found:
[ERROR]    javax.inject:javax.inject:jar:1:compile